frontend/sw360-portlet/src/main/webapp/html/projects/includes/projects/clearingStatus.jspf

```diff
@@ -66,24 +67,26 @@
             });
         });
 
-        function renderActions(releaseId) {
-            return "<img src='<%=request.getContextPath()%>/images/fossology-logo-24.gif' " +
-                " class='action send-to-fossology' data-release-id='" + releaseId + "'" +
-                "  alt='SelectClearing' title='send to Fossology'>";
-        }
-
         function createClearingTable() {
             var result = [];
 
             <core_rt:forEach items="${releasesAndProjects}" var="releasesClearingStatusData">
+            var sendToFossology = "<img src='<%=request.getContextPath()%>/images/fossology-logo-24.gif' " +
```

Comment:
One minor comment: The resulting JS code looks like:
```
       function createClearingTable() {
￼            var result = [];
            var sendToFossology = [...]
[...]
            var sendToFossology = [...]
[...]
       }
```

I do not know the implications of the multiple `var sendToFossology =` statements in one function, maybe extract add the line `var sendToFossology, editRelease;` to the top of the function and remove the superfluous `var ` statements.

Reply:
```suggestion
            sendToFossology = "<img src='<%=request.getContextPath()%>/images/fossology-logo-24.gif' " +
```